6/slim/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2-slim
+FROM python:3-slim
```

Comment:
I would love to use pyhton3, too. But it seems it is still not supported, please take a look at: 
* https://github.com/thumbor/thumbor/issues/1004
* https://github.com/thumbor/thumbor/issues/1035#issuecomment-358876296

---

I've tried it anyway and wasn't able to start the image:

```
$> docker run -p 8888:8888 thumbor-test-python-3
/usr/local/bin/python: No module named thumbor/server
```
Could you please revert this to python2?